klippa/src/lib.rs

```diff
@@ -343,6 +346,13 @@ impl Plan {
         this.populate_gids_to_retain(font);
         this.create_old_gid_to_new_gid_map();
 
+        //update the unicode to new gid list
+        let num = this.unicode_to_new_gid_list.len();
```

Comment:
is this related to the cblc/cbdt changes?

Reply:
no, but it's an important fix to avoid OTS failure